labs/lab0/chapter0.md

```diff
@@ -51,141 +49,80 @@ $ cd ~/
 $ git clone https://github.com/dustymabe/summit-2017-container-lab
 ```
 
-## Vagrant Walkthrough
-
-First off, if you are on a CentOS or RHEL host, enable the Vagrant 
-Software Collection:
+## Minishift Walkthrough
 
-```bash
-$ scl enable sclo-vagrant1 bash
-```
+Your major units of operation with minishift are `minishift start`, `minishift ssh`, 
+`minishift docker-env`, and `minishift stop`. We will walk through these. 
+Minishift has a number of other functions, some of which we will use later in the lab. However, these are the basics which warrant some examples to make sure you have enough context for the rest of the labs. We also need to get you access to the docker daemon running inside the minishift VM for the "Docker Refresh" in Lab 1.  
 
-Ok, so now you have Vagrant. Make sure by asking for the version (1.8.1 
-at time of writing):
-
-```bash
-$ vagrant --version
-Vagrant 1.8.1
-```
-
-Your major units of operation with Vagrant are `vagrant up`, `vagrant ssh`, 
-`vagrant halt`, and `vagrant destroy`. We will walk through these. 
-All of these operations, and almost every other Vagrant command, use a "per 
-project configuration file" called a `Vagrantfile` to define the details of 
-one or more virtual machines that you are using in your project. A detailed 
-explanation of a Vagrantfile and all the magic you can do with it is well 
-beyond the scope of this lab, but you can find out more details in the 
-[Documentation](https://www.vagrantup.com/docs/vagrantfile/).
-
-
-First, `vagrant up`: this command asks your hypervisor to launch the virtual 
-machine described in the Vagrantfile. The operation may be a "create and launch 
+First, `minishift start`: this command asks your hypervisor to launch the virtual 
+machine minishift has prepared. The operation may be a "create and launch 
 VM" or a "re-launch an existing VM" and it is largely transparent to the user. 
  
-OK, so, let's move in to a directory with a Vagrantfile and then launch the VM:
+OK, so, let's move in to our project directory and then launch minishift:
 
 ```bash
-$ cd ~/summit-2017-container-lab/vagrantcdk
-$ vagrant up
+$ cd ~/summit-2017-container-lab
+$ minishift start
 ```
 
 You should get a lot of feedback about the launch of the VM but, if you are 
 using the lab VM or have run this before, you will get a lot less. As long 
-as you don't get errors (delineated, normally, by red font) you are in good shape.
+as you don't get any errors you are in good shape.
 
-OK, so now you have a running VM. Ask vagrant to tell us the status:
+OK, so now minishift is running which means docker and OpenShift. We can now ask for the status (it's succinct!):
```

Comment:
`which means docker and OpenShift.....` do we need to add more to that sentence?